src/ui/FacetRange/FacetRange.ts

```diff
@@ -101,13 +102,15 @@ export class FacetRange extends Facet implements IComponentBindings {
   }
 
   public getValueCaption(facetValue: any): string {
```

Comment:
We should add unit tests for this method, checking the behaviour for different option combinations.